<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>681</width>
    <height>364</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTableView" name="tableView">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::SolidLine</enum>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Действия</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <widget class="QPushButton" name="addeventButton">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>30</y>
               <width>111</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>Добавить...</string>
             </property>
            </widget>
            <widget class="QPushButton" name="editeventButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>70</y>
               <width>111</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>Редактировать...</string>
             </property>
            </widget>
            <widget class="QPushButton" name="deleteeventButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>110</y>
               <width>111</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>Удалить</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Сортировка по</string>
            </property>
            <widget class="QComboBox" name="comboBox">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>30</y>
               <width>111</width>
               <height>31</height>
              </rect>
             </property>
             <item>
              <property name="text">
               <string>Кабинет</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Дата</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Время</string>
              </property>
             </item>
            </widget>
            <widget class="QCheckBox" name="showOccuredCheckBox">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>70</y>
               <width>121</width>
               <height>81</height>
              </rect>
             </property>
             <property name="text">
              <string>Показывать
прошедшие</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="tristate">
              <bool>true</bool>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="quitButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>111</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>Выход</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>698</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
